x/rewards/keeper/rewards.go

```diff
@@ -159,5 +159,5 @@ func (k Keeper) AllocationPoolMonthsLeft(ctx sdk.Context) int64 {
 		return math.MaxInt64
 	}
 
-	return int64(binary.BigEndian.Uint64(data[0]))
+	return int64(binary.BigEndian.Uint64(data[0])) // TODO: should we check data or trust binary package?
```

Comment:
Are you going to resolve this TODO?

Reply:
I checked it and forgot to delete. fixed https://github.com/lavanet/lava/commit/47cf05ecbab2e476059ef74e97c8c31f867fc287